src/pages/LandingPage/LandingPage.tsx

```diff
@@ -1,23 +1,36 @@
-import React, { useEffect } from "react";
+import React, { useEffect, useState } from "react";
 import VerticalNavigation from "../../shared/VerticalNavigation";
 import LandingHome from "./LandingHome";
 import LandingFeatures from "./LandingFeatures";
 import LandingTopics from "./LandingTopics";
 import Footer from "../../shared/Footer";
 import { Grid, Box, CircularProgress } from "@mui/material";
-import { useDispatch, useSelector } from "react-redux";
-import { RootState } from "../../app/common/store";
-import { requestModules } from "./LandingPageSlice";
+import apiClient from "../../app/common/apiClient";
+import { IModuleBase } from "../../shared/types";
 
 function LandingPage() {
-  const dispatch = useDispatch();
-  const isLoading = useSelector(
-    (state: RootState) => state.landingPage.isLoading
-  );
-  const modules = useSelector((state: RootState) => state.landingPage.modules);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [modules, setModules] = useState<string[]>([]);
   useEffect(() => {
-    dispatch(requestModules());
-  }, [dispatch]);
+    const fetchData = async () => {
```

Comment:
Think we're gonna have various calls like this throughout the app so it may be worth creating custom hooks for fetching different endpoints like this. Especially if we're reusing the same fetch in different pages

Reply:
created a useFetch hook with the url as the input. Let me know if this suffices